src/test/scala/karazin/scala/users/group/week2/homework/HomeworkSpecification.scala

```diff
@@ -50,28 +54,31 @@ object HomeworkSpecification extends Properties("Homework"):
   }
 
   property("negation") = forAll { (rational: Rational) =>
-    ???
+    rational.unary_- == Rational(-rational.numer, rational.denom) 
   }
 
   property("addition") = forAll { (left: Rational, right: Rational) =>

```

Comment:
This is a dirty test.
Why don't you compare the result `numer` with the sum of `numer`s?

Reply:
I'm not sure what do you mean by that

Reply:
Compare the result `numer` with the sum of `left` and `right` numers. The same for denoms.